src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  margin-top: 0;
+  font-size: 16px;
+}
+
+.input {
+  display: block;
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  padding: 0 0 0 62px;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  background: url(./images/Search.svg), #fff;
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  box-sizing: border-box;
+}
+
+.input--small {
+  font-size: 14px;
+  height: 42px;
+  line-height: 19px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 13px 15px;
+  background-size: 11px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 33px;
+}
```

Comment:
do not repeat the same code
write only those styles that differ in design from` .input`

and check it in whole file

Reply:
Okay. Do I need check my file and correct it?